src/main/resources/api/swagger.yaml

```diff
@@ -9,47 +9,53 @@ schemes:
 produces:
   - application/json
 paths:
-  /api/resolved-features:
-    get:
-      summary: Resolve all features by name
-      description: |
-        Returns all the features with all its dependencies in a single array.
-      tags:
-        - Features
-      parameters:
-        - name: names
-          in: query
-          description: feature names.
-          required: true
-          type: array
-          items:
-            type: string
-      responses:
-        200:
-          description: An array of features
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/Feature'
-        default:
-          description: Unexpected error
-          schema:
-            $ref: '#/definitions/Error'
   /api/features:
     get:
-      summary: Get all features
+      summary: Get features
       description: |
-        Returns all the features without its dependencies.
+        Returns the features.
       tags:
         - Features
       parameters:
         - name: names
           in: query
-          description: value, that must present in feature name.
+          description: exact names of the feature the call should return.
```

Comment:
I still think this should be `features` as this is probably returning more than one feature.